bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -193,3 +212,9 @@ message GetBookieInfoResponse {
 
 message StartTLSResponse {
 }
+
+message SyncResponse {
+    required StatusCode status = 1;
+    required int64 ledgerId = 2;
+    required int64 lastPersistedEntryId = 3;
```

Comment:
If we are using `lastAddSynced`, why can't we standardize the name as `lastAddSynced`?

Reply:
will do